www/js/util.js

```diff
@@ -87,15 +87,14 @@ function formatUserlistItem(div) {
         profile: div.data("profile") || { image: "", text: ""},
         leader: div.data("leader") || false,
         icon: div.data("icon") || false,
-        afk: div.data("afk") || false
     };
     var name = $(div.children()[1]);
     name.removeClass();
     name.css("font-style", "");
     name.addClass(getNameColor(data.rank));
     div.find(".profile-box").remove();
 
-    if (data.afk) {
+    if (div.data("meta") && div.data().meta.afk) {
```

Comment:
Why the inconsistency of using `.data("meta")` and `.data().meta` in the same line?